promgen/views.py

```diff
@@ -333,6 +333,19 @@ def post(self, request, pk):
             return redirect(sender.content_object)
         return redirect("profile")
 
+class ExporterUpdate(LoginRequiredMixin, UpdateView):
+    model = models.Exporter
+    button_label = _("Exporter Update")
+    form_class = forms.ExporterForm
+
+    def get_success_url(self):
+        return reverse('project-detail', args=[self.object.project_id])
+
+    def get_context_data(self, **kwargs):
+        context = super(ExporterUpdate, self).get_context_data(**kwargs)
+        context["project"] = self.object.project
```

Comment:
I think you might be able to replace this using `get_initial`

```
def get_initial(self):
    return {"project": self.object.project}
```

But I need to finish testing myself

Reply:
```ExporterRegister``` uses ```ProjectMixin``` to set ```project``` in the context.
https://github.com/line/promgen/blob/master/promgen/views.py#L608

Is it better to modify ```ExporterRegister``` to use ```get_initial``` as well?

Reply:
In that case, can probably use ProjectMixin the same way

Reply:
Since the path of ExporterUpdate  is ```/exporter/<exporter-pk>/edit```, I don't think we can use ProjectMixin in ExporterUpdate